server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/CassandraMailQueueViewConfiguration.java

```diff
@@ -67,6 +69,30 @@ public CassandraMailQueueViewConfiguration build() {
         return bucketCount -> updateBrowseStartPace -> sliceWindow -> new Builder.ReadyToBuild(bucketCount, updateBrowseStartPace, sliceWindow);
     }
 
+    public static CassandraMailQueueViewConfiguration from(Configuration configuration) {
+        Integer bucketCount = configuration.getInteger(BUCKET_COUNT_PROPERTY, null);
+        Preconditions.checkState(bucketCount != null, "You need to specify the " +
+            BUCKET_COUNT_PROPERTY + " property as the number of nodes from your cassandra cluster");
```

Comment:
No, not really. The buckets won't magically distribute accross the cluster evenly but rather randomly.

Thus one admin may wish to have more buckets in order to be sure of the repartition.

Exemple: With 3 node and 3 buckets I have a chance out of 27 to have all the buckets on the same node.

With one node and 9 buckets this is almost impossible.

I propose:

```
"You need to specify the " +
            BUCKET_COUNT_PROPERTY + " property. The higher the bucket count is, the better the data will be spread in the cluster. You might want the bucket count to exceed your Cassandra node count."
```